ggshield/cmd/auth/__init__.py

```diff
@@ -0,0 +1,8 @@
+import click
+
+from .login import login_cmd
```

Comment:
We are mixing relative and absolute imports a lot :(

Reply:
As discussed: I agree. I personally prefer absolute imports but sometimes relative imports are required to avoid circular import issues. This is often the case in `__init__.py` files.

But we can make another pass to fix this later.